### PR TITLE
Feature/bearc mpvmpr

### PR DESCRIPTION
**Pull request by bear-is-asleep**
Set the default values for sbnd's mpvmpr. I can't for the life of me get these changes to unstage so I'll just wait for them to be merged into develop to convert to a PR. 

* https://github.com/SBNSoftware/sbncode/pull/404 simple merge producer (and it's dependencies)

### CODE REVIEW

**Review by henrylay97 — COMMENTED** [review]
Looks good @bear-is-asleep, didn't fully understand your comment in the PR about the unstaging problem?

I've made a few comments but mainly aesthetic and can be ignored if you disagree with them.

**Review by PetrilloAtWork — CHANGES_REQUESTED** [review]
I can't vaucher on the values for SBND-specific parameters.
I would like to elicit a consideration: there are SBND-specific configurations here. These should be  moved to `sbndcode` or equivalent; especially `run_mpvmpr_sbnd.fcl` is a job configuration and that can't be run unless `sbndcode` is also pulled in (because of its dependence on `services_sbnd.fcl`).
I **recommend** the job configuration to be moved to `sbndcode`, and I **suggest** to move also the SBND-specific parameter settings in there (and yes, I know it's annoying to have another pull request).
You can choose to decline — anyway reply here with your intention.

File: sbncode/LArG4/MergeSimSourcesSBN_module.cc [hunk 1]
```diff
@@ -73,6 +73,18 @@ class sbn::MergeSimSourcesSBN : public art::EDProducer {
...
+      fhicl::Comment{"whether to merge MCParticleLite"},
+      true // default
+    };
+
+    fhicl::Atom<bool> FillMCParticlesAssociated{
+      fhicl::Name{"FillMCParticlesAssociated"},
+      fhicl::Comment{"whether to merge art::Assns<simb::MCParticle, simb::MCTruth, sim::GeneratedParticleInfo>"},
+      true // default
+    };
+
```
[the remark this answers is not in the document]
Comment (bear-is-asleep) [reply sbncode/LArG4/MergeSimSourcesSBN_module.cc]: sorry this is part of this [PR](https://github.com/SBNSoftware/sbncode/pull/397) which should've been merged already?

File: sbncode/LArG4/SimpleMerge_module.cc [hunk 2]
```diff
@@ -0,0 +1,91 @@
+////////////////////////////////////////////////////////////////////////
```
[the remark this answers is not in the document]
Comment (bear-is-asleep) [reply sbncode/LArG4/SimpleMerge_module.cc:1]: also part of a separate PR. I'm still overcoming my github noob phase!

**Comment by bear-is-asleep** [comment]
@henrylay97 @PetrilloAtWork Sorry some of the simple merge commits were merged into this code. I'm going to open a new PR just for the fcls, but will still take into considerations all of your comments and add you on those PRs. Thanks!

**Pull request closed without merging**